From 74bc238e86e62109c74d8f229dc105bf3818b4a7 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 2 Aug 2023 14:35:32 +0100
Subject: [PATCH 0823/1016] media: dt-bindings: imx258: Fix alternate
 compatible strings

Multiple compatible strings must appear as an enum.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 Documentation/devicetree/bindings/media/i2c/imx258.yaml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/media/i2c/imx258.yaml b/Documentation/devicetree/bindings/media/i2c/imx258.yaml
index 3fc30a79c538..a10dcdaf5e1d 100644
--- a/Documentation/devicetree/bindings/media/i2c/imx258.yaml
+++ b/Documentation/devicetree/bindings/media/i2c/imx258.yaml
@@ -19,8 +19,9 @@ description: |-
 
 properties:
   compatible:
-    const: sony,imx258
-    const: sony,imx258-pdaf
+    enum:
+      - sony,imx258
+      - sony,imx258-pdaf
 
   assigned-clocks: true
   assigned-clock-parents: true
-- 
2.42.0

